packages/driver-test-support/lib/e2e-suite.js

```diff
@@ -148,6 +149,11 @@ export function driverE2ETestSuite(DriverClass, defaultCaps = {}) {
 
     describe('session handling', function () {
       it('should handle idempotency while creating sessions', async function () {
+        if (util.compareVersions(process.version, '>=', '19.0.0')) {
```

Comment:
```suggestion
        if (util.compareVersions(process.version, '>=', '20.0.0')) {
```

v19 is unmaintained

Reply:
it is, but I saw people are still using it. Sol just in case of cases...